scripts/aem.js

```diff
@@ -100,62 +116,6 @@ async function loadFonts() {
   }
 }
 
-/**
- * Add <img> for icon, prefixed with codeBasePath and optional prefix.
- * @param {span} [element] span element with icon classes
- */
-function decorateIcon(elem) {
-  if (elem.dataset.decorated) return;
-
-  const iconName = Array.from(elem.classList)
-    .find((c) => c.startsWith('icon-'))
-    .substring(5);
-  const img = document.createElement('img');
-  img.dataset.iconName = iconName;
-  img.src = `${window.hlx.codeBasePath}/icons/${iconName}.svg`;
-  img.loading = 'lazy';
-  elem.append(img);
-
-  elem.dataset.decorated = true;
-}
-
-/**
- * Decorates paragraphs containing a single link as buttons.
- * @param {Element} element container element
- */
-function decorateButton(a) {
-  if (a.dataset.decorated) return;
-
-  a.title = a.title || a.textContent;
-
-  if (a.href !== a.textContent) {
-    const up = a.parentElement;
-    const twoup = a.parentElement.parentElement;
-
-    if (a.querySelector('img')) return;
-
-    if (
-      up.childNodes.length === 1
-      && up.tagName === 'STRONG'
-      && twoup.childNodes.length === 1
-      && twoup.tagName === 'P'
-    ) {
-      a.className = 'button primary';
-    }
-
-    if (
-      up.childNodes.length === 1
-      && up.tagName === 'EM'
-      && twoup.childNodes.length === 1
-      && twoup.tagName === 'P'
-    ) {
-      a.className = 'button secondary';
-    }
-
-    a.dataset.decorated = true;
-  }
-}
```

Comment:
Moved decorators to `arm-root` where they can be customize on the project level.

Reply:
Also, this was causing to run multiple times since it was in the root and every brick. Having it just in the root is enough.